gradle.properties

```diff
@@ -1,18 +1,18 @@
-# Project-wide Gradle settings.
+ariaTenantTokenProd="b6618d4b5d24466a9d2d0d8cde8cd6e4-ddcec952-23e9-4765-b4fb-f823ec69c6e3-7966"
+ariaTenantTokenTest="1e8435186fa849b28b1a402fb5074ff1-0b91a9ec-efad-440c-b92c-ac4cb55ba0ff-7490"
 
-# IDE (e.g. Android Studio) users:
-# Gradle settings configured through the IDE *will override*
-# any settings specified in this file.
+android.useAndroidX=true
+android.enableJetifier=true
 
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
+# https://office.visualstudio.com/Outlook%20Mobile/_wiki/wikis/Outlook-Mobile.wiki/3780/Android-Studio-Gradle-Performance-tips-and-tricks
+org.gradle.parallel=true
+org.gradle.daemon=true
 
-# Specifies the JVM arguments used for the daemon process.
-# The setting is particularly useful for tweaking memory settings.
-# Default value: -Xmx10248m -XX:MaxPermSize=256m
-org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+# See https://stackoverflow.com/questions/56075455/expiring-daemon-because-jvm-heap-space-is-exhausted
+org.gradle.jvmargs=-Xmx4g -XX:MaxPermSize=2048m -Dkotlin.daemon.jvm.options\="-Xmx2048M" -XX:ReservedCodeCacheSize=512m
 
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
+# This the TSL versionName...
+versionName=1.5.9
+
+# For OneAuth default abiSelection
+abiSelection=x86_64
```

Comment:
Should we have these changes checked in?

Reply:
These [are taken from `dev`](https://github.com/AzureAD/azure-activedirectory-library-for-android/blob/dev/gradle.properties) - [needed for desugaring support](https://github.com/AzureAD/azure-activedirectory-library-for-android/commit/426acaafb53e0c23c06882a4b8752403e08a379e)